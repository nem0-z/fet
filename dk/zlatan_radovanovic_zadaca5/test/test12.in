foo_ = bar1  = 3 * 3.14 / 180
print foo_
print bar1

