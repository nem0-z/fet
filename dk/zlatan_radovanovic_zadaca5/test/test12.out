0.052333
0.052333
